docs/contributtion/experimental-code.md

```diff
@@ -0,0 +1,172 @@
+any changes to them across our releases.
+
+#### Flags prefixed with `enable-v#-*`
+
+As usage of an existing flag increases or we determine a feature to be of high
+importance, we'll "commit" it to a future major release and rename it to use the
+`enable-v#-*` prefix. At this point the API or functionality behind this flag is
+now fixed and won't change. We intend to ship this flag as "on by default" in
+the major version indicated in the name. e.g. `enable-v3-some-feature`
+
+All breaking changes will be shipped as `enable-v#-*` flags within the current
+releases. This enables projects to opt-in to breaking changes earlier
+and at their own pace avoiding one huge changeset when upgrading to the next
+major release. In theory, if all `enable-v#-*` flags are enabled within your
+project before the major release, no changes should need to be made when updating
+to next major version.
+
+For a flag to be committed to a release and renamed to `enable-v#-*` it must:
+
+- Be tested with early adopters
+- Be fully covered in tests (Unit, Visual)
+- Be documented in Storybook
+- Be documented on https://spirit.design
+- Have an automated migration script (codemod) available, where possible
+
+### Documenting Feature Flags
+
+Within the experimental section of the Spirit Design System Storybook, there is a
+"Feature Flags" folder. This contains stories and written documentation covering
+every available flag and how to configure them. Folders and stories within this
+section in the Storybook show components with all feature flags turned on.
+
+All the currently available feature flags, as well as how to enable them, is
+documented in the main package README file.
+
+Individual feater flags that are available for some component only will be also documented in
+the component's README file.
+
+## Expanding existing APIs
+
+New experimental functionality might sometimes be added with the need of an
+`UNSTABLE_` export or a feature flag. Oftentimes this ends up being a new prop
+on a component.
+The documentation associated with this prop (TypeScript types, Storybook controls, etc.) will state that it's experimental.
+
+## Moving to stable
+
+Over time it becomes apparent an experimental API has stabilized and suits the
+needs of most users. If it isn't a breaking change, and there hasn't been much
+movement, it can be moved from "experimental" status to be "stable".
+
+To initiate a move to stable, a new issue can be opened to request the team to evaluate
+if it can be moved (most of the time the team will initiate this move).
+The issue should contain the following criteria that need to
+be met to move a component from experimental to stable:
+
+- [ ] All components exported in `src/index.ts` and should not be `UNSTABLE_`
+      prefixed
+- [ ] Component should be documented on the website
+  - [ ] Component should have a usage, style, and code tab
+  - [ ] Component may have a component demo
+- [ ] For each component exported:
```

Comment:
maybe mention that this is only for react?

Reply:
Fixed.